matchms/exporting/save_spectra.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pickle
+from typing import List
+from matchms.exporting import save_as_json, save_as_mgf, save_as_msp
+from matchms.Spectrum import Spectrum
+from matchms.typing import SpectrumType
+
+
+def save_spectra(spectrums: List[SpectrumType],
+                 file: str) -> None:
+    """Saves spectra as the file type specified.
+
+    The following file extensions can be used:
+    "json", "mgf", "msp"
+
+    Args:
+    -----
+    spectrums:
+        The spectra that are saved.
+    file:
+        Path to file containing spectra, with file extension "json", "mgf", "msp"
+    ftype:
+        Optional. Filetype
+    """
+    assert not os.path.exists(file), f"The specified file: {file} does already exist"
```

Comment:
```suggestion
    if os.path.exists(file):
        raise FileExistsError(f"The specified file: {file} already exists.")
```

Reply:
Assert should not be used in this way. One reason is seen in the suggested change which raises a more specific error. The second is that assert will not be run if Python is invoked with optimization. Biopython had the same issue https://github.com/biopython/biopython/issues/3600

Reply:
Thanks @fabianegli Before, I never really understood why assert should not be used in production code. Thanks for clarifying!